### PR TITLE
Fixes the taste of the moss dumplings

### PR DESCRIPTION
**Pull request by FlamingLily**
They no longer taste like meat and flatbread

### CODE REVIEW

**Comment by FlamingLily** [comment]
!review

**Comment by FlamingLily** [comment]
!bugfix

File: html/changelogs/flaminglily-patch-1.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - bugfix: "Fixed the flavour of moss dumplings."
```
Comment (SleepyGemmy) [inline html/changelogs/flaminglily-patch-1.yml:41]:
```suggestion
  - bugfix: "Fixes the flavour of moss dumplings."
```

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 2]
```diff
@@ -4688,8 +4688,8 @@
 	name = "moss dumplings"
 	desc = "A relatively common Konyanger dish, this appears to be steamed moss set in steamed dough."
 	icon_state = "moss_dumplings"
-	reagents_to_add = list(/decl/reagent/nutriment = 3, /decl/reagent/nutriment/protein = 3)
-	reagent_data = list(/decl/reagent/nutriment = list("flatbread" = 3))
+	reagents_to_add = list(/decl/reagent/nutriment = 3, /decl/reagent/nutriment/moss = 3)
+	reagent_data = list(/decl/reagent/nutriment = list("steamed dough" = 3))
```
Comment (SleepyGemmy) [inline code/modules/reagents/reagent_containers/food/snacks.dm:4692]:
```suggestion
	reagent_data = list(/decl/reagent/nutriment = list("steamed dough" = 3, "moss" = 3))
```

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

**Pull request merged**